utils/build/docker/java/spring-boot/src/main/java/com/datadoghq/system_tests/springboot/App.java

```diff
@@ -287,29 +287,15 @@ String raspSSRF(@RequestParam(required = false, name="url") String url) {
     @EventListener(ApplicationReadyEvent.class)
     @Trace
     public void init() {
+        new AppReadyHandler(this).start();
+        /*
         cassandra = new CassandraConnector();
```

Comment:
If we don't need this code, let's remove it, not comment it out.

Reply:
Done